module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -191,33 +163,6 @@ public function addListTag($tagText, $user)
         }
     }
 
-    /**
-     * Saves the properties to the database.
-     *
-     * This performs an intelligent insert/update, and reloads the
-     * properties with fresh data from the table on success.
-     *
-     * @param \VuFind\Db\Row\User|bool $user Logged-in user (false if none)
-     *
-     * @return mixed The primary key value(s), as an associative array if the
-     *     key is compound, or a scalar if the key is single-column.
-     * @throws ListPermissionException
-     * @throws MissingFieldException
-     */
-    public function save($user = false)
```

Comment:
I deleted this because of the way it interferes with the normal operation of the Laminas\Db framework. While there's something to be said for preventing unwanted saves from occurring at such a low level, it's not practical to carry that behavior forward.